$NetBSD: patch-ar,v 1.4 2024/04/09 01:12:36 pho Exp $

Fix build on NetBSD: struct iovec resides in <sys/uio.h>

--- lib/include/iovector.h.orig	2024-02-06 15:08:41.000000000 +0000
+++ lib/include/iovector.h
@@ -37,7 +37,7 @@ extern "C" {
  * Ugly definition of struct iovec.
  */
 #if defined(__linux__) || defined(sun) || defined(__APPLE__) || \
-    defined(__FreeBSD__) || defined(__EMSCRIPTEN__)
+    defined(__FreeBSD__) || defined(__NetBSD__) || defined(__EMSCRIPTEN__)
 #include <sys/uio.h>    // for struct iovec
 #else
 
